modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -346,6 +348,20 @@ def international_phone_too_long?
         phone.length > 25 if phone
       end
 
+      def validate_form_526_gulf_service!
+        gulf_war_service = form_attributes&.dig('toxicExposure', 'gulfWarHazardService')
+        return if gulf_war_service&.dig('servedInGulfWarHazardLocations') == 'NO'
+
+        gulf_regex = '^(?:19|20)[0-9][0-9]$|^(?:19|20)[0-9][0-9]-(0[1-9]|1[0-2])$'
```

Comment:
Wondering if this would be better utilized as a constant in the file since the YYYY and YYYY-MM formats will likely be utilized in other areas? Especially if the REGEX patterns are going to be moved out of the 526.json file and into here.  

I know it is kind of nitpicky, but a variable name that would more clearly convey what it is doing might be better here too, especially if it does work to make this a constant for checking date these date patterns.